app/src/components/utils.js

```diff
@@ -53,6 +56,41 @@ const utils = {
       if (!/^\w+$/.test(k)) throw new Error(`Invalid field name (${k}) in \`context\`.  Only alphanumeric characters and underscore allowed.`);
     });
     return true;
+  },
+
+  validateAttachments: (attachments, attachmentLimit = '5mb') => {
+    if (attachments) {
+      if (!Array.isArray(attachments)) {
+        throw new Error('Invalid value `attachments`');
+      } else {
```

Comment:
need it because attachments aren't required, and without a schema, devs could just leave the  field out. so only checking the field if included in the body.